src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -232,7 +238,7 @@ private String getSubmissionFormTableHeaderFragmentHtml(String questionNumberStr
     }
 
     private String getSubmissionFormTableBodyHtml(String questionNumberString, String responseNumberString,
-            boolean sessionIsOpen, boolean isExistingResponse, FeedbackRubricResponseDetails frd) {
+             boolean sessionIsOpen, boolean isExistingResponse, FeedbackRubricResponseDetails frd) {
```

Comment:
For this one align `boolean sessionIsOpen` with `String questionNumberString`. Break to one more line if needed, it should still be acceptable (not too squeezed to the right).